README.md

```diff
@@ -15,3 +15,36 @@ contains two packages:
 ```
 pip install suitcase-mongo
 ```
+
+## Quick Start
+
+`suitcase-mongo` supports `mongo_normalized` and `mongo_embedded`.
+
+Using `mongo_normalized`:
+
+```
+from suitcase.mongo_normalized import Serializer
+docs = db[-1].documents(fill=True)
+serializer = Serializer(METADATASTORE_DB, ASSET_REGISTRY_DB)
+for item in documents:
+  serializer(*item)
+```
+
+Using `mongo_embedded`:
+
+```
+from suitcase.mongo_normalized import Serializer
+import event_model
+
+docs = db[-1].documents(fill=True)
+serializer = Serializer(PERMANENT_DB)
+for item in docs:
+  # Fix formatting for JSON.
+  item = event_model.sanitize_doc(item)
```

Comment:
I don't think we need this line or the comment. Also it doesn't appear in the normalized version

Reply:
https://github.com/bluesky/suitcase-mongo/blob/master/suitcase/mongo_embedded/__init__.py#L181

Reply:
ok